tidb-cloud/features.md

```diff
@@ -0,0 +1,278 @@
+  </tr>
+  <tr>
+    <td rowspan="3" style="background-color: white;">Cluster management</td>
+    <td>Pause & Resume</td>
+    <td style="text-align:center;">❌</td>
+    <td style="text-align:center;">❌</td>
+    <td style="text-align:center;">✅</td>
+  </tr>
+  <tr>
+    <td>Maintenance window</td>
+    <td style="text-align:center;">❌</td>
+    <td style="text-align:center;">❌</td>
+    <td style="text-align:center;">✅</td>
+  </tr>
+  <tr>
+    <td>Recycle bin</td>
+    <td style="text-align:center;">❌</td>
+    <td style="text-align:center;">❌</td>
+    <td style="text-align:center;">✅</td>
+  </tr>
+  <tr>
+    <td rowspan="2" style="background-color: white;">Data import</td>
+    <td>Import (using <code>IMPORT INTO</code> or the TiDB Cloud console)</td>
+    <td style="text-align:center;">✅</td>
+    <td style="text-align:center;">✅</td>
+    <td style="text-align:center;">✅</td>
+  </tr>
+  <tr>
+    <td>Data migration</td>
+    <td style="text-align:center;">❌</td>
+    <td style="text-align:center;">✅ <br/><span style="font-size: 14px; white-space: nowrap;">(Private preview)</span></td>
+    <td style="text-align:center;">✅</td>
+  </tr>
+  <tr>
+    <td rowspan="2" style="background-color: white;">Data export</td>
+    <td>Export</td>
+    <td style="text-align:center;">✅ <br/><span style="font-size: 14px; white-space: nowrap;">(Public preview)</span></td>
+    <td style="text-align:center;">✅ <br/><span style="font-size: 14px; white-space: nowrap;">(Public preview)</span></td>
+    <td style="text-align:center;">❌</td>
+  </tr>
+  <tr>
+    <td>Changefeed</td>
+    <td style="text-align:center;">❌</td>
+    <td style="text-align:center;">✅ <br/><span style="font-size: 14px; white-space: nowrap;">(Private preview in CLI)</span></td>
+    <td style="text-align:center;">✅</td>
+  </tr>
+  <tr>
+    <td rowspan="4" style="background-color: white;">Backup &amp; restore</td>
+    <td>Automatic backup</td>
+    <td style="text-align:center;">✅</td>
+    <td style="text-align:center;">✅</td>
+    <td style="text-align:center;">✅</td>
+  </tr>
+  <tr>
+    <td>Dual region backup</td>
+    <td style="text-align:center;">❌</td>
+    <td style="text-align:center;">❌</td>
+    <td style="text-align:center;">✅</td>
+  </tr>
+  <tr>
+    <td>Manual backup</td>
+    <td style="text-align:center;">❌</td>
+    <td style="text-align:center;">❌</td>
+    <td style="text-align:center;">✅</td>
+  </tr>
+  <tr>
+    <td>Point-in-time recovery (PITR)</td>
+    <td style="text-align:center;">❌</td>
+    <td style="text-align:center;">✅</td>
+    <td style="text-align:center;">✅</td>
+  </tr>
+  <tr>
+    <td style="background-color: white;">Storage</td>
+    <td>Standard storage</td>
+    <td style="text-align:center;">❌</td>
+    <td style="text-align:center;">❌</td>
+    <td style="text-align:center;">✅</td>
+  </tr>
+  <tr>
+    <td rowspan="7" style="background-color: white;">Observability</td>
+    <td>In-app metrics</td>
```

Comment:
```suggestion
    <td>Built-in metrics</td>
```

Reply:
@likidu Can we use "Built-in metrics" to align with the wording in docs https://docs.pingcap.com/tidbcloud/built-in-monitoring/?